relay/kusama/src/lib.rs

```diff
@@ -3180,7 +3217,9 @@ mod multiplier_tests {
 		let mut fees_paid = 0;
 
 		frame_system::Pallet::<Runtime>::set_block_consumed_resources(Weight::MAX, 0);
-		let info = DispatchInfo { weight: Weight::MAX, ..Default::default() };
+		// TODO: Find out if this test is correct, since we're not yet considering
+		// `extension_weight`
+		let info = DispatchInfo { call_weight: Weight::MAX, ..Default::default() };
```

Comment:
Test is still good, info here is only used to track the fees for such weight, and log it. Whether the weight is for the call or the extension doesn't matter.